Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp

```diff
@@ -495,9 +495,8 @@ bool WKBundlePageFindString(WKBundlePageRef pageRef, WKStringRef target, WKFindO
     return WebKit::toImpl(pageRef)->findStringFromInjectedBundle(WebKit::toWTFString(target), WebKit::toFindOptions(findOptions));
 }
 
-void WKBundlePageFindStringMatches(WKBundlePageRef pageRef, WKStringRef target, WKFindOptions findOptions)
+void WKBundlePageFindStringMatches(WKBundlePageRef, WKStringRef, WKFindOptions)
```

Comment:
If there were programs that needed to keep finding this in the framework (like SafariForWebKitDevelopment on old OSes) then we would use WK_C_API_DEPRECATED and keep this here.  Since this is only used in WebKitTestRunner and has been for years, let's just remove it.